app.js

```diff
@@ -0,0 +1,47 @@
+// import functions and grab DOM elements
+import { getRandomThrow } from './r-p-s.js';
+import { checkResult } from './r-p-s.js';
+
+const playButton = document.getElementById('play-button');
+const gameResults = document.getElementById('results');
+const currentTotal = document.getElementById('current-total');
+
+// initialize state
+let win = 0;
+let draw = 0;
+let lose = 0;
+let totalGuesses = 0;
+
+// set event listeners to update state and DOM
+playButton.addEventListener('click', () => {
+    console.log('hello');
+
+    //if total guesses is 3, end game
+    if (totalGuesses === 4) {
+        return;
+    }
+
+    //generate random number
+    getRandomThrow();
+
+    //get user input
+    const userInput = document.querySelector('input:checked');
+    const userGuess = userInput.value;
+    console.log(userGuess);
+
+    currentTotal.textContent = totalGuesses++;
+    console.log(totalGuesses);
+
+    //compare inputs & add points
+    if (checkResult() === 'draw') {
```

Comment:
Your `checkResult` function needs a computerGuess and a userGuess to able to compare and provide a return. Without providing checkResults with arguements, it will always comeback undefined and none of these if conditions will be met.